dali/pipeline/operator/op_schema.cc

```diff
@@ -115,29 +115,47 @@ std::string OpSchema::GetArgumentDox(const std::string &name) const {
   DALI_ENFORCE(HasArgument(name), "Argument \"" + name +
       "\" is not supported by operator \"" + this->name() + "\".");
   if (HasRequiredArgument(name)) {
-    return GetRequiredArguments().at(name).first;
+    return GetRequiredArguments().at(name).doc;
   } else {
     // optional argument
-    return GetOptionalArguments().at(name).first;
+    return GetOptionalArguments().at(name).doc;
   }
 }
 
 DALIDataType OpSchema::GetArgumentType(const std::string &name) const {
   DALI_ENFORCE(HasArgument(name), "Argument \"" + name +
       "\" is not supported by operator \"" + this->name() + "\".");
   if (HasRequiredArgument(name)) {
-    return GetRequiredArguments().at(name).second;
+    return GetRequiredArguments().at(name).dtype;
   } else {
     // optional argument
-    return GetOptionalArguments().at(name).second->GetTypeID();
+    return GetOptionalArguments().at(name).dtype;
   }
 }
 
+bool OpSchema::HasArgumentDefaultValue(const std::string &name) const {
+  DALI_ENFORCE(HasArgument(name, true), "Argument \"" + name +
```

Comment:
make_string ?

Reply:
I left it aligned with other warnings in this files.